00_Base/src/mapper/session.mapper.ts

```diff
@@ -201,7 +201,7 @@ export class SessionMapper {
             });
             const previousEnergy =
               this.getEnergyImportForMeterValue(previousMeterValue);
-            if (previousEnergy) {
+            if (previousEnergy !== undefined) {
```

Comment:
In this case, `previousEnergy` was coming back as 0 since the energy import of the previous charging period (the first charging period) was 0.

Therefore, this condition was returning false.